assignment/src/com/CodingQuestions/Question17/UserMainCode.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class UserMainCode {
+     public static int findMaxDistance(int[] a)
+    {
+        Scanner s=new Scanner(System.in);
+        int n=s.nextInt();
+        for(int i=0;i<n;i++)

```

Comment:
If 'n' is >20, we will get ArrayIndexOutOfBoundsException. Hence, better to take 'n' as 20 directly.